migrations/20170911221051-add-osInfo-processInfo-version-to-collector.js

```diff
@@ -0,0 +1,27 @@
+'use strict'; // eslint-disable-line strict
+const TBL = 'Collectors';
+
+module.exports = {
+  up(qi, Sequelize) {
+    return qi.sequelize.transaction(() =>
+      qi.addColumn(TBL, 'osInfo', {
+        type: Sequelize.JSONB,
+        allowNull: true,
+      })
+      .then(() => qi.addColumn(TBL, 'processInfo', {
+        type: Sequelize.JSONB,
+        allowNull: true,
+      }))
+      .then(() => qi.addColumn(TBL, 'version', {
+        type: Sequelize.STRING,
+        allowNull: false,
```

Comment:
does migration work if adding a column with `allowNull: false` if there are already records in the db? might have to provide default value (if so, maybe just set default value to `0.0.0` to imply we don't have a version yet)?

Reply:
We had the default version set as 1.0.0 for SGT. Wondering if we want to do the same for the collectors too.?

Reply:
default ver for SGT is different ... here we're just making sure nothing breaks if a record existed before this column was added